.vscode/settings.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Why are you deleting it?

Reply:
I thought this was user-specific config for VS Code. Reverted.